utoipa-gen/src/openapi/info.rs

```diff
@@ -41,7 +41,15 @@ impl Info<'_> {
                     Some(contact)
                 }
             });
-        let license = std::env::var("CARGO_PKG_LICENSE").ok().map(License::from);
+        let license = std::env::var("CARGO_PKG_LICENSE")
```

Comment:
I'd perhaps just update the `From<T>` implementation for License to support the `info.license.identifier` as well.

Reply:
According to [the openapi specification](https://swagger.io/specification/), the `name` of the `license` object is a *name* of the license. For example `Apache 2.0`. And it's may not a SDPX license identifier.

If the `string` argument of `License::from()` is a SDPX license expression, it's ok to set the `string` value to the `name` and `identifier`. However, if the `string` argument is a license name, its value should not be set to the `identifier`.

Reply:
modified in 4c501ce97105e23a99bc43168e8845fa16942961

Reply:
True :+1: